dags/catalyst.py

```diff
@@ -0,0 +1,46 @@
+"""
+DAG to schedule generation of performance reports for recently completed nimbus experiments.
+
+See the [jetstream repository](https://github.com/mozilla/catalyst).
+
+"""
+
+from datetime import datetime, timedelta
+
+from airflow import DAG
+
+from operators.gcp_container_operator import GKEPodOperator
+from utils.tags import Tag
+
+default_args = {
+    "owner": "dpalmeiro@mozilla.com",
+    "email": [
+        "dpalmeiro@mozilla.com",
+    ],
+    "depends_on_past": False,
+    "start_date": datetime(2025, 3, 18),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 1,
+    "retry_delay": timedelta(minutes=30),
+}
+
+tags = [Tag.ImpactTier.tier_1]
```

Comment:
Related to the above comment on Airflow Triage, just wanted to make sure you're aware of the `no_triage` and `record_only` tags that will tell the triage person to skip the DAG entirely, or only record a bug without attempting to investigate. I don't think you need either of these, but FYI in case you think they're applicable.